Microsoft.Dynamics365.UIAutomation.Api.UCI/DTO/AppElementReference.cs

```diff
@@ -492,7 +492,7 @@ public static class AppElements
             { "Entity_Header_DateTimeFieldContainer","//div[@data-id='header_[NAME]-FieldSectionItemContainer']" },
                         
             //CommandBar
-            { "Cmd_Container"       , "//ul[contains(@data-lp-id,\"commandbar-Form\")]"},
+            { "Cmd_Container"       , ".//ul[contains(@data-lp-id,\"commandbar-Form\")]"},

```

Comment:
Can you confirm that the following methods still function with this change?

- CommandBar.GetCommandValues();
- CommandBar.ClickCommand();

Reply:
yes this also works I am alreday using this at both places